easybuild/easyconfigs/f/FastQC/FastQC-0.10.1-Java-1.7.0_80.eb

```diff
@@ -17,22 +17,23 @@ description = """FastQC is a quality control application for high throughput seq
  application to review the results of several different QC checks, or create an HTML based
  report which can be integrated into a pipeline."""
 
-toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 sources = ['%(namelower)s_v%(version)s.zip']
 source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
 
 java = 'Java'
 javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver)]
 
-dependencies = [('Java', '1.7.0_80', '', True)]
-
-install_cmd = "cp -a FastQC/* %(installdir)s && chmod +x %(installdir)s/fastqc"
+postinstallcmds = ['chmod +x %(installdir)s/fastqc']
```

Comment:
Well, no, but I then wonder why you changed the `sanity_check_paths` too; I assumed you had to do that because of this change, but if `PackedBinary` copies everything, there's no reason to change `sanity_check_paths`?